docs/knowledge-submissions-past-wikipedia.md

```diff
@@ -42,7 +42,7 @@ Status:
 ### Next steps
 
 1. We have to find the correct legal person to find a way to be the correct point person for this project.
-1. Collect suggested places from the community and add them to the above table
+1. Collect [suggested data source domains from the community](https://github.com/instructlab/dev-docs/issues/59) and add them to the above table.
```

Comment:
This implies the issue will be open "forever" as a receiving site for suggestions. Perhaps a github discussion might be better for this role?

Reply:
We keep going back and forth on what we plan to use GitHub discussion for vs. issues. I think perhaps the real place to collect this info is in a thread on the community mailing list. Thoughts?